Tasks/PublishTestResultsV2/make.json

```diff
@@ -2,7 +2,7 @@
   "externals": {
     "archivePackages": [
       {
-        "url": "https://publishtestresults.blob.core.windows.net/publishtestresults/8068203/PublishTestResults.zip",
+        "url": "https://publishtestresults.blob.core.windows.net/publishtestresults/8568741/PublishTestResults.zip",
```

Comment:
task.json and loc.json. update the version

Reply:
Done. loc.json version is already 148